controllers/platformnetwork_controller.go

```diff
@@ -835,7 +835,17 @@ func (r *PlatformNetworkReconciler) ReconcileResource(client *gophercloud.Servic
 			return err
 		}
 		if instance.Status.DeploymentScope == cloudManager.ScopeBootstrap {
```

Comment:
We only want to ignore configuration of oam/mgmt/admin networks, should allow the other types

Reply:
Updated the code with  check for network type